contribute.json

```diff
@@ -0,0 +1,38 @@
+{
+  "name": "Phabricator",
+  "description": "Mozilla's Phabricator instance for code review",
+  "repository": {
+    "url": "https://github.com/mozilla-services/mozphab",
+    "license": "Apache License, Version 2.0",
+    "tests": "https://circleci.com/gh/mozilla-services/mozphab/tree/master"
+  },
+  "participate": {
+    "home": "https://wiki.mozilla.org/Phabricator",
+    "docs": "https://moz-conduit.readthedocs.io/en/latest/phabricator-user.html",
+    "mailing-list": "https://www.mozilla.org/en-US/about/forums/#tools",
+    "irc": "irc://irc.mozilla.org/#phabricator",
+    "irc-contacts": [
+      "mcote"
+    ]
+  },
+  "bugs": {
+    "list": "https://bugzilla.mozilla.org/buglist.cgi?resolution=---&component=Phabricator%20%28Upstream%29&component=Phabricator%20Extensions&product=Conduit&list_id=13774015",
```

Comment:
Omit "list_id" param (it is used to cache a bug list, and we want to make sure this link always displays the currently open bugs).